### ISSUE TITLE
Bug: mailtarget error exePath must be specified when not running inside a stand alone exe

### ISSUE THREAD

**Issue by ericnewton76** [NLog/NLog#1493]
Bug: using MailTarget, getting error "exePath must be specified when not running inside a stand alone exe."

```
[ArgumentException: exePath must be specified when not running inside a stand alone exe.]
   System.Configuration.ConfigurationManager.OpenExeConfigurationImpl(ConfigurationFileMap fileMap, Boolean isMachine, ConfigurationUserLevel userLevel, String exePath, Boolean preLoad) +687332
   NLog.Targets.MailTarget.get_MailSettings() +37
```

Running in an ASPNet MVC app, obviously shouldn't be using OpenExeConfiguration in MailTarget.getMailSettings

**NLog version**: 4.3.2

**Platform**: .Net 4.5 / Azure

**Current NLog config** (xml or C#, if relevant)

```xml
<?xml version="1.0" encoding="utf-8" ?>
<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
      xsi:schemaLocation="http://www.nlog-project.org/schemas/NLog.xsd NLog.xsd"
      autoReload="true"
      throwExceptions="false"
      internalLogLevel="Off" internalLogFile="c:\temp\nlog-internal.log" >


	<!-- optional, add some variabeles
  https://github.com/nlog/NLog/wiki/Configuration-file#variables
  -->
	<variable name="myvar" value="myvalue"/>

	<!-- 
  See https://github.com/nlog/nlog/wiki/Configuration-file 
  for information on customizing logging rules and outputs.
   -->
	<targets>

		<!-- 
    add your targets here 
    See https://github.com/nlog/NLog/wiki/Targets for possible targets.
    See https://github.com/nlog/NLog/wiki/Layout-Renderers for the possible layout renderers.
    -->

		<!--
    Writing events to the a file with the date in the filename. 
    <target xsi:type="File" name="f" fileName="${basedir}/logs/${shortdate}.log"
            layout="${longdate} ${uppercase:${level}} ${message}" />
    -->
		<target xsi:type="File" name="fileTarget" fileName="${basedir}\App_Data\logs\${shortdate}.log"
			layout="${date:universalTime=true:format=yyyy-MM-dd hh\:mm\:ss.fff}|${uppercase:${level}}|${logger}||${message} ${exception:format=tostring}" />
		<target xsi:type="Debugger" name="debuggerTarget" 
			layout="${date:format=yyyyMMddTHHmmss.fff}|${level:uppercase=true}|${logger}|${message} ${exception:format=tostring}" />
		<target xsi:type="Mail" name="errorMailTarget"
			smtpServer="xxxxxxxxx.net" smtpPort="587" enableSsl="true" smtpUserName="xxxxxxxxxx@xxxxxx.com" smtpPassword="xxxxxxxxxx" 
			layout="${longdate} ${uppercase:${level}} ${logger} ${message} ${exception:format=tostring}"
			from="YYYYYY@yyy.com"
			to="XXXXXXXXX@xxxxx.com"
			subject="Error occurred"
			addNewLines="true"
		/>
	</targets>

	<rules>
		<!-- add your logging rules here -->

		<!--
    Write all events with minimal level of Debug (So Debug, Info, Warn, Error and Fatal, but not Trace)  to "f"
    <logger name="*" minlevel="Debug" writeTo="f" />
    -->
		<logger name="*" minlevel="Error" writeTo="errorMailTarget" />
		<!--<logger name="*" minlevel="Trace" writeTo="debuggerTarget" />-->
		<logger name="*" minlevel="Info" writeTo="fileTarget" />
		
	</rules>
</nlog>

```

**Comment by 304NotModified** [comment]
Do you have a stacktrace?

**Comment by ericnewton76** [comment]
Just realized this is a duplicate of issue #1407

**Comment by ericnewton76** [comment]
@304NotModified stack trace is second paragraph

**Comment by 304NotModified** [comment]
Thanks,

This is indeed already fixed and released in NLog 4.3.3
